Microsoft.Azure.Devices.Edge.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
-VisualStudioVersion = 15.0.27130.0
+# Visual Studio Version 16
```

Comment:
Probs a non issue but are you aware of this version bump? Will this break those using VS 2017?